packages/libraries/core/src/components/file/file-item/ods-file-item-controller.ts

```diff
@@ -0,0 +1,25 @@
+import { OdsFileItem } from './ods-file-item';
+import { OdsComponentController } from '../../ods-component-controller';
+
+export class OdsFileItemController extends OdsComponentController<OdsFileItem> {
+  // private readonly logger = new OdsLogger('OdsFileController');
+
+  constructor(component: OdsFileItem) {
+    super(component);
+  }
+
+  /**
+   * Attributes validation documentation
+   */
+  validateAttributes(): void {
+    if (!this.component.name) {
+      throw new Error('ods-file-item: name attribute is required');
+    }
+    if (!this.component.size) {
+      throw new Error('ods-file-item: size attribute is required');
+    }
+  }
+  getSizeInKiloBytes(size: number) {
+    return Math.round(size / 1000);
```

Comment:
Could be great to handle the NaN case also (as you have some for example in the e2e)